pybuildkite/client.py

```diff
@@ -1,6 +1,8 @@
 import requests
 from urllib.parse import urlparse
 
+from pybuildkite.constants import APIVersion
+
 
 class Client(object):
```

Comment:
So this is really a client for the Buildkite API, not so much the Agent API. If we want to include Agent API it probably makes sense to refactor this to have two separate clients.

Reply:
Done. I tried my best to preserve current functionality, even for internal implementations. There are a number of breaking changes I would suggest, but IMO, are better in a follow up since this change is already becoming large.

Reply:
The follow up would look like, https://github.com/jmelahman/pybuildkite/pull/2 `Breaking changes from pull#80`